src/hrflow_connectors/connectors/boards/smartrecruiters/actions.py

```diff
@@ -0,0 +1,178 @@
+            .get("text")
+        )
+        additional_information = (
+            data.get("jobAd", {})
+            .get("sections", {})
+            .get("additionalInformation", {})
+            .get("text")
+        )
+        job["sections"] = [
+            dict(
+                name="company_description",
+                title="companyDescription",
+                description=companyDescription,
+            ),
+            dict(
+                name="job_description",
+                title="jobDescription",
+                description=jobDescription,
+            ),
+            dict(
+                name="qualifications", title="qualifications", description=qualification
+            ),
+            dict(
+                name="additional_information",
+                title="additionalInformation",
+                description=additional_information,
+            ),
+        ]
+        # job tags
+        status = data.get("status")
+        posting_status = data.get("postingStatus")
+        job_uuid = data.get("id")
+        experience_level = data.get("experienceLevel", {}).get("id")
+        employmentType = data.get("typeOfEmployment", {}).get("id")
+        industry = data.get("industry", {}).get("id")
+        creator = data.get(
+            "creator",
+        )
+        function = data.get("function", {}).get("id")
+        department = data.get("department", {}).get("id")
+        manual = data.get("location", {}).get("manual")
+        remote = data.get("location", {}).get("remote")
+        eeo_category = data.get("eeoCategory", {}).get("id")
+        compensation = data.get("compensation", {})
+        job["tags"] = [
+            dict(name="status", value=status),
+            dict(name="posting_status", value=posting_status),
+            dict(name="job_uuid", value=job_uuid),
+            dict(name="experience_level", value=experience_level),
+            dict(name="type_of_employment", value=employmentType),
+            dict(name="compensation", value=compensation),
+            dict(name="industry", value=industry),
+            dict(name="creator", value=creator),
+            dict(name="function", value=function),
+            dict(name="department", value=department),
+            dict(name="manual", value=manual),
+            dict(name="remote", value=remote),
+            dict(name="eeo_category", value=eeo_category),
+        ]
+        # ranges of duration and salary
+        job["ranges_date"] = [
+            dict(
+                name="targetHiringDate",
+                value_min=None,
+                value_max=data.get("targetHiringDate"),
+            )
+        ]
+        job["ranges_float"] = [
+            dict(
+                name="compensation",
+                value_min=data.get("compensation", {}).get("min"),
+                value_max=data.get("compensation", {}).get("max"),
+                unit=data.get("compensation", {}).get("currency"),
+            )
+        ]
```

Comment:
In reality, few people use `ranges_date` and `ranges_float`. That's why I put them all in tags or metadata. Especially since you have to make sure that `value_min < value_max` otherwise it won't be indexed in the Board and it will create a lot of problems. That's why I prefer to put them in less restrictive fields and convert them to String :warning: .